Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml

```diff
@@ -72,7 +72,7 @@
     modifiers:
       coefficients:
         Heat: 0.90
-        Radiation: 0.05
```

Comment:
undocumented?

Reply:
Rad hoods did literally nothing before, because for some reason old mesons were functioning as rad hoods while rad hoods were purely cosmetic. I didn't really notice until I removed the mesons from the radiation closet

Reply:
it doesn't really matter too much as the pr is small, but unrelated balance changes should really either be listed in the pr description and changelog or moved to another pr altogether.

Reply:
this is nerfing radsuits/hoods overall though? it used to be 0.05 * (whatever mesons had) resist but now its 0.45 * 0.65 resist which is way worse

Reply:
just remove this change and keep the added rad hood resist